_data/toc.yaml

```diff
@@ -1400,12 +1400,6 @@ manuals:
     title: Docker Scan
   - path: /engine/sbom/
     title: Docker SBOM (Experimental)
-
-- sectiontitle: Docker Build
-  section:
-    - path: /build/
-      title: Overview
-
```

Comment:
Why as this been removed?

Reply:
Good catch. Sorry, didn't recognize this change after the last commit. @dockertopia Could we bring this in again? Thanks.

Reply:
This will be merged to Published probably before. Or is Build section already on the master ready to be published? 
I understood that was going later on when everyone was good on a first version for Docker Build to be public.

Reply:
@crazy-max Does this make sense to you? My understanding is that this change in TOC will not affect a page/section that does exist in master yet?

Reply:
> Does this make sense to you?

I'm just surprised by this change in this PR because it's out of scope with its description. Would have been better for another PR so we can track this easily.

> I understood that was going later on when everyone was good on a first version for Docker Build to be public

Ok I misunderstood, I thought we wanted to iterate the changes and make them public as we go along.